common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -8,7 +8,7 @@
 <form class="choicegroup capa_inputtype" id="inputtype_${id}">
     <fieldset ${describedby}>
       % if response_data['label']:
-        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
+        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${HTML(response_data['label'])}</legend>
```

Comment:
I think you could probably use bleach, at least to get the attribute values.  You could also use bleach to ensure safe HTML, depending on if you know what tags you want to support.

Here's some sample code using bleach:
```
>>> import bleach
>>> bleach.ALLOWED_TAGS
[u'a', u'abbr', u'acronym', u'b', u'blockquote', u'code', u'em', u'i', u'li', u'ol', u'strong', u'ul']
>>> bleach.clean("<li>hello</li>")
u'<li>hello</li>'
>>> bleach.clean("<li>hello</li>", tags=[])
u'&lt;li&gt;hello&lt;/li&gt;'
>>> bleach.clean("<p>hello</p>")
u'&lt;p&gt;hello&lt;/p&gt;'
>>> bleach.clean("<li>hello & goodbye</li>", tags=[], strip=True)
u'hello &amp; goodbye'
```
You probably want something like the last line for attribute values.  This may sounds strange, but since the output of bleach is HTML, you'd need something like:
```
import bleach
aria-label="${HTML(bleach.clean(response_data['label'], tags=[], strip=True)}"
```
For places where you want real HTML, you'll need:
```
${HTML(response_data['label'])}  # assumes instructor will only add safe HTML
```
Or
```
${HTML(bleach.clean(response_data['label']))}  # only accepts limited HTML
```
Here are [additional docs for bleach](http://bleach.readthedocs.io/en/latest/clean.html) if you need them.